extensions/omni/test/omni_test.c

```diff
@@ -136,6 +136,12 @@ void _Omni_init(const omni_handle *handle) {
                                   .fn = {.xact_callback = xact_callback}};
   handle->register_hook(handle, &xact_callback_hook);
 
+  omni_hook planner_hook = {.name = "planner_hook",
+                            .type = omni_hook_planner,
+                            .fn = {.planner = planner_hook_fn},
+                            .wrap = true};
```

Comment:
I'm not sure whether setting `wrap` to true is required here, as I understand `wrap` will add the same hook to the beginning and end of list of a particular hook type, an example usecase of `wrap` would help to decide when to use it.